### PR TITLE
allow BAM to BED to consume more diverse BAM datatypes

### PR DESCRIPTION
**Pull request by bgruening**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/bedtools/bamToBed.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@ $split
 > '${output}'
     ]]></command>
     <inputs>
-        <param name="input" type="data" format="bam" label="Convert the following BAM file to BED"/>
+        <param name="input" type="data" format="bam,qname_input_sorted.bam,qname_sorted.bam,unsorted.bam" label="Convert the following BAM file to BED"/>
```
Comment (mvdbeek) [inline tools/bedtools/bamToBed.xml:28]:
```suggestion
        <param name="input" type="data" format="unsorted.bam" label="Convert the following BAM file to BED"/>
```
That corresponds to the BamNative class which all the other bam classes inherit from.

Comment (bgruening) [reply tools/bedtools/bamToBed.xml:28]: Oh good to know. And I guess I should know, damn! Danke.

**Review by bernt-matthias — APPROVED** [review]
Bump?

**Comment by bgruening** [comment]
Puh I don't know, it sould not change anything in reruns. You call :)

**Comment by bernt-matthias** [comment]
> Puh I don't know, it sould not change anything in reruns. You call :)

I'm fine with it. @mvdbeek any further comments? Otherwise I would merge.

**Comment by mvdbeek** [comment]
It's not just the reruns, if you allow additional inputs and you have the old version on one server, and the new version on another server you will run a different number of jobs if an input is unsorted if you run the workflow on both instances. And what if we made a mistake ? Debugging would be more difficult then ... that's a long way of saying I almost always prefer version bumps.

**Review by mvdbeek — APPROVED** [review]

**Pull request merged**